website_sale_stock_available_display/__manifest__.py

```diff
@@ -4,18 +4,20 @@
     "name": "Website Sale Stock Available Display",
     "summary": "Display stock in shop online and allow to sell with no stock "
     "available",
-    "version": "12.0.1.2.1",
+    "version": "13.0.1.0.0",
     "development_status": "Beta",
     "category": "Website",
     "website": "https://github.com/OCA/e-commerce",
     "author": "Tecnativa, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "application": False,
     "installable": True,
-    "depends": ["website_sale_stock",],
+    "depends": ["website_sale_stock"],
+    "demo": ["data/demo.xml"],
     "data": [
         "views/assets.xml",
+        "templates/assets.xml",
         "views/product_template_views.xml",
```

Comment:
Why is this file empty?

Reply:
It isn't. Check it https://github.com/OCA/e-commerce/pull/504/files#diff-9c89c8dcae270794ccd62d9d8dea916a4ab9dd47da380053317c624998f51b7eR1-R20